src/modules/AutoinstGeneral.rb

```diff
@@ -489,6 +515,22 @@ def AutoinstGeneral
       nil
     end
 
+  protected
+
+    # Return the value to use as 'btrfs_set_default_subvolume_name' in the profile
+    #
+    # In case it matches the product's default, it should not be exported.
+    #
+    # @return [Boolean,nil] Value to use (true or false). If nil, no value
+    #                       should be exported.
+    def btrfs_default_subvol_to_profile
+      if FileSystems.default_subvol != FileSystems.default_subvol_from_product
```

Comment:
so in fact it depends on storage-team to make necessary changes there. Good to know. Thanks for inspecting it.